### ISSUE TITLE
Editing forces view to automatically scroll unjustly

### ISSUE THREAD

**Issue by gima** [jbt/markdown-editor#22]
It's useful to see "the full picture" of how the output looks. This new feature makes it hard, as it forcibly scrolls the edited portion into the top of the view pane.

It would be nice if the scrolling didn't happen when the text that's being edited is already in view. It's perfectly reasonable to scroll the edited portion into view when it's not visible.

**Comment by jbt** [comment]
Agreed - the current logic for scrolling is as in #12 which I think is definitely better than not scrolling at all. I'll have another look and see if it can be improved at all. I think using `scrollIntoViewIfNeeded` might improve things somewhat.
